In [1]:
import numpy as np

# Задание1.

Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз

ФАКТР(100)/(ФАКТР(85)*ФАКТР(15))*0,8^85*0,2^15

Вероятность равна 0,048061794


# Задание 2.

Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек. Какова вероятность, что ни одна из них не перегорит в первый день? Какова вероятность, что перегорят ровно две?

$Pm≈λ^m/m!*e^-λ$

In [2]:
n = 5000
m = 0
p = 0.0004
lambda_ = n * p

(lambda_**m / np.math.factorial(m)) * np.exp(-lambda_)

0.1353352832366127

Вероятность, что ни одна из них не перегорит в первый день  13,5%

In [4]:
n = 5000
m = 2
p = 0.0004
lambda_ = n * p

(lambda_**m / np.math.factorial(m)) * np.exp(-lambda_)

0.2706705664732254

Вероятность, что ни одна из них не перегорит в первый день 27,07%

# Задание 3.

Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

$Pn(X=k)=Cknp^kq^(n−k)$

ФАКТР(144)/(ФАКТР(70)*ФАКТР(74))*0,5^70*0,5^74 = 0,06281178

Вероятность равна 0,06281178

# Задание 4.

В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. Какова вероятность того, что все мячи белые? Какова вероятность того, что ровно два мяча белые? Какова вероятность того, что хотя бы один мяч белый?

### 4.1.

In [9]:
(7/10*6/9)*(9/11*8/10)

0.30545454545454537

вероятность того, что все мячи белые 0,305454545

### 4.2.

In [6]:
(7/10)*(6/9)*(2/11)*(1/10) + (7/10)*(3/9)*(9/11)*(2/10) + (7/10)*(3/9)*(2/11)*(9/10) + \
(3/10)*(7/9)*(9/11)*(2/10) + (3/10)*(7/9)*(2/11)*(9/10) + (3/10)*(2/9)*(9/11)*(8/10)

0.20484848484848484

вероятность того, что ровно два мяча белые 0.2048

### 4.3. 

In [5]:
1-(3/10*2/9*2/11*1/10)

0.9987878787878788

вероятность того, что хотя бы один мяч белый  0.999